src/parser/jobs/blm/modules/RotationWatchdog.js

```diff
@@ -212,16 +223,30 @@ export default class RotationWatchdog extends Module {
 			}))
 		}
 
-		//Suggestion for hard T3s under AF. Will be enabled as soon as T3Ps stop being dumb
-		if (this._wrongT3) {
+		// Suggestion to end Astral Fires with Despair
+		if (this._astralFiresNotEndedWithDespair) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.DESPAIR.icon,
+				content: <Trans id="blm.rotation-watchdog.suggestions.end-with-despair.content">
+					Casting <ActionLink {...ACTIONS.BLIZZARD_III} /> to enter Umbral Ice costs no MP. Always end Astral Fire with a <ActionLink {...ACTIONS.DESPAIR} /> to make full use of your MP.
+				</Trans>,
+				severity: SEVERITY.MAJOR,
+				why: <Trans id="blm.rotation-watchdog.suggestions.end-with-despair.why">
+					<Plural value={this._astralFiresNotEndedWithDespair} one="# Astral Fire phase" other="# Astral Fire phases"/> ended with a spell other than <ActionLink {...ACTIONS.DESPAIR} />.
+				</Trans>,
+			}))
+		}
+
+		//Suggestion for hard T3s under AF. Should only have one per cycle
+		if (this._extraT3s) {
 			this.suggestions.add(new Suggestion({
 				icon: ACTIONS.THUNDER_III_FALSE.icon,
 				content: <Trans id="blm.rotation-watchdog.suggestions.wrong-t3.content">
-					Never hard cast a <ActionLink {...ACTIONS.THUNDER_III}/> in your Astral Fire phase, since that costs MP which could be used for more <ActionLink {...ACTIONS.FIRE_IV}/>s.
+					Don't hard cast more than one <ActionLink {...ACTIONS.THUNDER_III}/> in your Astral Fire phase, since that costs MP which could be used for more <ActionLink {...ACTIONS.FIRE_IV}/>s.
 				</Trans>,
 				severity: SEVERITY.MAJOR,
```

Comment:
Should this always be major? Seems a bit harsh.

Reply:
Hardcasting more than one T3 in a fire cycle means you're going to lose out on either a Fire 4 or a Despair due to MP issues, which is the same situation you'd be in prior to ShB for hardcasting any at all. Since we were dinging Major then, I think it's still appropriate (also with the sharpcast change, you have so many opportunities for Thunderclouds now, you REALLY shouldn't be hardcasting it twice in fire)

Reply:
There is no reaason to hardcast T3 in af twice since your dot is long enough to last into UI. It was a major before and should be major now. Dots are 5% of our damage, F4s are about 75%